exporter/stackdriverexporter/stackdriver.go

```diff
@@ -154,6 +156,21 @@ func (me *metricsExporter) pushMetrics(ctx context.Context, m pdata.Metrics) (in
 	for _, md := range mds {
 		d, err := me.mexporter.PushMetricsProto(ctx, md.Node, md.Resource, md.Metrics)
 		dropped += d
+
+		var points int
+		var st string
+		if err != nil {
+			points = d
+			s, ok := status.FromError(err)
+			if ok {
+				st = statusCodeToString(s)
+			}
+		} else {
+			_, points = pdatautil.TimeseriesAndPointCount(md)
+			st = "OK"
+		}
+		recordPointCount(ctx, points, st)
```

Comment:
Do we also want to report on the number of dropped points here?

Reply:
This code does the following:

- If an error was returned, set points = dropped points (as returned by `PushMetricsProto`) & status = gRPC error status code; this may be less than the total number of points in the request if it was a partial failure
- If no error was returned, set points = total number of points in the request & status = OK
- Now that I think about it, for partial failures, we should record the number of points that succeeded and I'm totally missing that here 😮 

Let me fix point 3 and try to make it a bit clearer in follow-up PR